proxy/src/control/discovery.rs

```diff
@@ -462,10 +427,10 @@ where
         for (auth, set) in &mut self.destinations {
             // Query the Destination service first.
             let (new_query, found_by_destination_service) = match set.query.take() {
-                Some(DestinationServiceQuery::ConnectedOrConnecting{ rx }) => {
+                Some(Remote::ConnectedOrConnecting{ rx }) => {
```

Comment:
TIL we can't refer to this as `DestinationServiceQuery` because of https://github.com/rust-lang/rust/issues/26264. (No change necessary.)